### PR TITLE
first pr for ticket 10

### PR DESCRIPTION
**Pull request by EddySafc**

### CODE REVIEW

File: models/model.js [hunk 1]
```diff
@@ -9,7 +9,13 @@ exports.fetchCategories = () => {
...
     .query(
-      "SELECT owner, title, reviews.review_id, category, review_img_url, reviews.created_at, reviews.votes, designer, COUNT(comments.comment_id) AS comment_count FROM reviews LEFT JOIN comments ON comments.review_id = reviews.review_id GROUP BY reviews.review_id ORDER BY reviews.created_at DESC;"
+      `SELECT owner, title, reviews.review_id, category, review_body, review_img_url, reviews.created_at, reviews.votes, designer, COUNT(comments.comment_id) 
+      AS comment_count 
+      FROM reviews 
+      LEFT JOIN comments 
+      ON comments.review_id = reviews.review_id 
+      GROUP BY reviews.review_id 
+      ORDER BY reviews.created_at DESC;`
     )
```
Comment (cjplatten) [inline models/model.js:19]: This ticket is for a single review by id, not all reviews.

File: controllers/controller.js [hunk 2]
```diff
@@ -30,9 +30,16 @@ exports.getReviews = (req, res, next) => {
...
     .then((result) => {
-      res.send(result);
+      result.forEach((review) => {
+        if (review.review_id === parseInt(review_id)) {
+          res.send(review);
+        }
+      });
+      if (parseInt(review_id) == review_id) {
+        return Promise.reject({ status: 404, msg: "id not found" });
+      } else return Promise.reject({ status: 400, msg: "bad request" });
```
Comment (cjplatten) [inline controllers/controller.js]: why has this code been changed for this ticket? This ticket is for the individual review, not all reviews.

**Review by cjplatten — COMMENTED** [review]
Please review comments.

File: controllers/controller.js (hunk 2 above)
Comment (EddySafc) [reply controllers/controller.js]: I've used fetchReviews in the model to return back an array of all the reviews to the controller (including the comment_count) and then used the review_id parameter to pick out the single review from this array to then be sent.

**Pull request merged**